client/src/modules/list/components/Items/ItemsList.jsx

```diff
@@ -26,19 +26,19 @@ class ItemsList extends PureComponent {
   }
 
   componentDidMount() {
-    this.handleItems();
+    this.prepareDisplayedItems();
   }
 
   componentDidUpdate(previousProps) {
     const { items: previousItems } = previousProps;
     const { items } = this.props;
 
     if (previousItems.length !== items.length) {
-      this.handleItems();
+      this.prepareDisplayedItems();
     }
   }
 
-  handleItems = () => {
+  prepareDisplayedItems = () => {
```

Comment:
handleItemsToDisplay

Reply:
I think handle is more for handle an action from user input, button press etc.